script/update-json-schema.sh

```diff
@@ -0,0 +1,4 @@
+NEW_HASH=$(git ls-remote git://github.com/department-of-veterans-affairs/vets-json-schema/ refs/heads/master | awk '{print $1}')
+sed -i -e 's/\(vets-json-schema\.git#\)\(.*\)\(",$\)/\1'$NEW_HASH'\3/' package.json
+rm package.json-e
```

Comment:
The sed command creates a `package.json-e` file that then needs to be cleaned up. I'm not sure why this file gets created :-/ Anyone know how to prevent that so the cleanup isn't needed?

Reply:
That file isn't created for me locally, it seems like something set up in your environment is causing this.

Reply:
Weird. Adding an empty string after the `-i` flag prevents it from happening on my machine. I'm not sure why it acts differently for us.